README.md

```diff
@@ -8,34 +8,83 @@ Rosetta is maintained by the RosettaCommons, a collaboration of 50+ academic res
+
+``` sh
+git clone https://github.com/RosettaCommons/rosetta
+```
+and change to that directory
+``` sh
+cd rosetta
+```
 
 Getting Started Using Rosetta
-============================
+=============================
 
 Start here: https://www.rosettacommons.org/docs/latest/getting_started/Getting-Started
 
-Compilation quick-start (see <https://www.rosettacommons.org/docs/latest/build_documentation/Build-Documentation> for more details):
+The fastest way to address your scientific modeling challenge at hand may be with the official Docker images.
+Accessible from the official Docker hub at https://hub.docker.com/r/rosettacommons/rosetta,
+the images have both Rosetta and PyRosetta pre-installed, such that Rosetta tutorials can be followed.
+
+Anticipating that you like what you see or if coming with a Computer Science background,
+you may decide to extend and compile the Rosetta source tree yourself. See the
+Compilation quick-start on <https://www.rosettacommons.org/docs/latest/build_documentation/Build-Documentation> for more details.
+
+The Rosetta source tree ships with all its run-time dependencies, just when building you
+need to install the C++ compiler (g++ or clang). Also you need the scripting language Python to be installed,
+The compilation is then performed by:
 
 ``` sh
-$ cd Rosetta/main/source
+$ cd source
 $ ./scons.py -j<NumOfJobs> mode=release bin
 ```
 
 Questions about how to use Rosetta are best directed to the RosettaCommons forums <https://www.rosettacommons.org/forum>
+The Rosetta source tree is big and uses a series of advanced features of the C++ language.
+Some C++ compilers may not yet perfectly master these features.
+Later versions of those compilers have that fixed.
+Talk back to us when you run into issues.
 
 PyRosetta
 =========
 
-PyRosetta are Python bindings to the Rosetta library. These can be built from the Rosetta source code. 
+PyRosetta are Python bindings to the Rosetta library. These can be built from the Rosetta source code.
 
 See <https://www.pyrosetta.org/> for more information about PyRosetta.
+For details on the building and installation from the source tree refer to
+<https://www.pyrosetta.org/documentation#h.q91xbiqnd5yv>.
+
+The Docker image referenced above already ships with PyRosetta.
+To prepare the PyRosetta Python module locally from this source tree,
+you need
+ * a C++ compiler, like the one you used to compile the other parts of Rosetta
+ * the Ninja ([conda](https://anaconda.org/conda-forge/ninja), [Debian](https://tracker.debian.org/pkg/ninja-build)) build management tool 
+ * and also CMake ([conda](https://anaconda.org/conda-forge/cmake), [Debian](https://tracker.debian.org/pkg/cmake))
+which should all be readily available from your regular Linux distribution.
+
+``` sh
+$ cd source/src/python/PyRosetta
+$ python3 build.py -j24 --create-package $HOME/my_pyrosetta_package
+$ cd $HOME/my_pyrosetta_package/setup
+$ python3 setup.py install
```

Comment:
I have related PR https://github.com/RosettaCommons/rosetta/pull/41/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32 which try to fix the same things. 

@roccomoretti what is your take on this? One way to do this would be to merge source/src/python/PyRosetta/README.md and this README files or we can keep PyRosetta readme separate at source/src/python/PyRosetta/README.md - thoughts?

Reply:
I was not asked, please forgive: Since PyRosetta is a project on GitHub in its own right, and likely so for more than only historic reasons, IMHO it needs a README that can stand for itself.  I have not yet thought too much over what the PyRosetta README I would expect to present, but if you consider this exchange over this README.md fruitful then I would come up with a likely similar PR also for the PyRosetta README.md.